crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1242,7 +1246,7 @@ impl RpcServerConfig {
                             http_cors_domains: Some(http_cors.clone()),
                             ws_cors_domains: Some(ws_cors.clone()),
                         }
-                        .into())
+                        .into());
```

Comment:
this was done by my ide autofmt on save - ill try and take some time to revert these in the next commit